scw/client.go

```diff
@@ -90,6 +91,10 @@ func defaultOptions() []ClientOption {
 	return []ClientOption{
 		WithAPIURL("https://api.scaleway.com"),
 		WithUserAgent(userAgent),
+		WithPaginationConfig(&PaginationConfig{
+			Full:     false,
```

Comment:
What is the exact purpose of `full`? If it's true the pagination is disabled?
Maybe we could find a better name